src/config.js

```diff
@@ -67,6 +67,7 @@ let config = {
   LOGO_TRADEMARK_URL: process.env.LOGO_TRADEMARK_URL,
   LOGO_WHITE_URL: process.env.LOGO_WHITE_URL,
   FAVICON_URL: process.env.FAVICON_URL,
+  MFE_CONFIG_API_URL: process.env.MFE_CONFIG_API_URL,
```

Comment:
I think this value needs to be added in a few more places.  As a way to discover places it should appear, I tried searching for `LOGO_TRADEMARK_URL` above and noted that config values in this file are also set in:

- `.env.test`
- `.env.development` 
- `setupTest.js`

Basically, it needs to be defaulted in those files for completeness sake.

Reply:
I could add the key without value by default because is optional  :)